src/pybind/mgr/dashboard/frontend/src/app/ceph/block/mirroring/bootstrap-create-modal/bootstrap-create-modal.component.ts

```diff
@@ -102,18 +109,16 @@ export class BootstrapCreateModalComponent extends BaseModal implements OnDestro
           ++checkedCount;
         }
       });
-
+      this.generateBtnDisable = poolsControl.status == 'DISABLED';
```

Comment:
Avoid string literals:
```suggestion
      this.generateBtnDisable = poolsControl.status == POOL_DISABLED_STATUS;
```